controllers/mapController.js

```diff
@@ -110,7 +115,87 @@ const addInvitedUser = async (req, res, next) => {
   }
 };
 
+const inviteNewMember = async (req, res, next) => {
+  const { id } = req.params;
+  const { email } = req.body;
+
+  try {
+    const currentMap = await Map.findById(id).exec();
+    const newUser = await User.findOne({ email }).exec();
+
+    const userId = newUser._id;
+    const verifyInvitedEmail = currentMap.invitationList
+      .map(user => user.email)
+      .some(invitedEmail => invitedEmail === email);
+
+    if (!newUser) {
+      res.json({
+        result: "등록되지 않은 유저입니다.",
+      });
+      return;
+    }
```

Comment:
newUser가 있는지 없는지 판단하기 전에 userId를 먼저 할당해주면 안될 것 같은 생각이 들어요..!
초대한 이메일을 가진 유저가 없다면 objectId도 undefined가 될테니까요..!

newUser라는 네이밍은 뭔가 회원가입처럼 새로운 유저를 등록해주는 느낌이 들어서 invitedUser 같은 네이밍은 어떨까 싶습니다..!

```suggestion
    const currentMap = await Map.findById(id).exec();
    const newUser = await User.findOne({ email }).exec();
    
    if (!newUser) {
      res.json({
        result: "등록되지 않은 유저입니다.",
      });
      return;
    }

    const userId = newUser._id;
    const verifyInvitedEmail = currentMap.invitationList
      .map(user => user.email)
      .some(invitedEmail => invitedEmail === email);
```

Reply:
헉 맞습니다! 감사합니다~!